### ISSUE TITLE
Can we have easing options exposed by IAnimationOptions 

### ISSUE THREAD

**Issue by arnoldsimha** [filipows/angular-animations#49]
Hi,
I really like this library. It allows us to focus on development and less on animations development. There is one thing I would like to have and this is an easing option that will allow the developers to decide the easing for the animations.
Thank you

**Comment by filipows** [comment]
Hi @arnoldsimha, 
thank you for your feedback! I might have to look closer into that, to not over-complicate things. Do you have any particular use case in mind or it's more of a general thought? 

Some of the animations are having different easing functions between keyframes, so should we expose all of the easing functions for a certain animation? (in some cases it could be 3 functions per animation)

**Comment by arnoldsimha** [comment]
Hi @filipows ,
Thank you for your response. I think it's possible to keep it simple. I saw that you're setting "easing" property as "ease":
![image](https://user-images.githubusercontent.com/815961/81486307-8557ed00-925c-11ea-8710-18f50790cc68.png)

you can expose it through IAnimationOptions and allow developers to set the string, or even have list of allowed options :)

**Comment by mckenzielong** [comment]
Was also wondering if we could get this feature. It would be nice to achieve the emphasized easing mentioned in the material design reference:

https://material.io/design/motion/speed.html#easing

**Comment by yousafraza39142** [comment]
> Hi @filipows ,
> Thank you for your response. I think it's possible to keep it simple. I saw that you're setting "easing" property as "ease":
> ![image](https://user-images.githubusercontent.com/815961/81486307-8557ed00-925c-11ea-8710-18f50790cc68.png)
> 
> you can expose it through IAnimationOptions and allow developers to set the string, or even have list of allowed options :)

Yes I was also looking for easing parameter but couldn't find one. Could you suggest any work around

**Comment by filipows** [comment]
Hi @yousafraza39142,

thanks for your comment. Unfortunately, I don't think there's a workaround for that atm.

Seems like this feature should be prioritised. I'll have a look at this in the upcoming days.
